--- origsrc/wxGlade-0.6.3/Makefile	2008-02-01 09:20:36.000000000 -0600
+++ src/wxGlade-0.6.3/Makefile	2008-06-30 02:39:48.984375000 -0500
@@ -19,7 +19,7 @@
 	cp -a *.py codegen edit_sizers res widgets \
 	  $(DESTDIR)/usr/lib/python$(PYVER)/site-packages/wxglade
 	# fix executable flags
-	for f in configUI.py zwxglade.py; do \
+	for f in configUI.py wxglade.py; do \
 	  chmod 755 $(DESTDIR)/usr/lib/python$(PYVER)/site-packages/wxglade/$$f; \
 	done
 	for f in edit_widget.py config.py; do \
@@ -31,7 +31,8 @@
 	find $(DESTDIR) -name '.cvsignore' -type f | xargs rm -f
 	ln -s /usr/share/$(PACKAGE)/icons \
 	  $(DESTDIR)/usr/lib/python$(PYVER)/site-packages/wxglade
-	install -m 755 wxglade $(DESTDIR)/usr/bin
+	sed -e "s,@PYVER@,$(PYVER),g" wxglade.in > $(DESTDIR)/usr/bin/wxglade
+	chmod 755 $(DESTDIR)/usr/bin/wxglade
 install-doc: debian/wxglade.1
 	gzip -c9 debian/wxglade.1 > $(DESTDIR)/usr/share/man/man1/wxglade.1.gz
 	cp -a docs $(DESTDIR)/usr/share/doc/$(PACKAGE)/
--- origsrc/wxGlade-0.6.3/wxglade.in	1969-12-31 18:00:00.000000000 -0600
+++ src/wxGlade-0.6.3/wxglade.in	2008-06-30 02:39:48.984375000 -0500
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec python@PYVER@ /usr/lib/python@PYVER@/site-packages/wxglade/wxglade.py
